exasol-jdbc/src/main/scala/com/exasol/spark/rdd/ExasolRDD.scala

```diff
@@ -110,14 +109,6 @@ class ExasolRDD(
         case e: Exception => logWarning("Received an exception closing sub resultSet", e)
       }
 
-      try {
-        if (stmt != null && !stmt.isClosed) {
-          stmt.close()
-        }
-      } catch {
-        case e: Exception => logWarning("Received an exception closing sub statement", e)
-      }
-
       try {
```

Comment:
👍

Reply:
Do you plan to update version number, changelog, dependencies?

Reply:
Not at the moment - there are couple of small issues I'd like to fix before the release